source/ed/data.h

```diff
@@ -99,6 +99,8 @@ class Data: boost::noncopyable {
 
     std::vector<ed::types::AdminStopArea*>  admin_stop_areas;
 
+    std::vector<ed::types::ObjectProperty*> object_properties;
```

Comment:
I checked out, and I understand that the situation is the same for almost all ed types vectors (like admin_stop_areas)
So as I am new to Navitia, I supposed that as "fusio2ed" is a short lived script the memory leeks are considered of no importance.
I have coded the same with std::vector<ed::types::ObjectProperty>, but that will made a recopy of the object when push_back in object_properties.
So this is as you want : 
1) I let the code unchanged because memory leak is accepted in that context
2) I change it to a simple object
3) I change it to unique_ptr

But in case 2 and 3, someone must do the same change on all ed::data vectors.
You ? Me ?